source/admin_manual/list_of_events.rst

```diff
@@ -1650,6 +1650,9 @@ dns_request_finished
 +---------------------+------------------------------------------------------+
 | Field               | Description                                          |
 +=====================+======================================================+
+| cached              | Set to ``yes`` or ``no`` depending if it was         |
+|                     | a cached reply or not.                               |
```

Comment:
versionadded needed also.

Reply:
should know what version to use though =)

Reply:
didn't we decide 3.0?..